lib/periodicChecks.js

```diff
@@ -0,0 +1,168 @@
+// Copyright 2020 The Oppia Authors. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS-IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+/**
+ * @fileoverview Handler for checks to be carried out periodically.
+ */
+
+const {
+  getAllOpenPullRequests,
+  hasPullRequestBeenApproved,
+  getLastReviewOfType,
+  doesPullRequestHaveChangesRequested,
+} = require('./utils');
+const mergeConflictModule = require('./checkMergeConflicts');
+const pullRequestReviewModule = require('./checkPullRequestReview');
+const { teamLeads } = require('../userWhitelist.json');
+const CHANGES_REQUESTED = 'changes_requested';
+const { oppiaMaintainers } = require('../userWhitelist.json');
+
+/**
+ *  This function checks if a pull request has merge conflicts.
+ *
+ * @param {import('probot').Octokit.PullsGetResponse} pullRequest
+ */
+const hasMergeConflict = (pullRequest) => {
+  return pullRequest.mergeable === false;
+};
+
+/**
+ *  This function checks if a pull request has pending reviews.
+ *
+ * @param {import('probot').Octokit.PullsGetResponse} pullRequest
+ */
+const hasPendingReviews = (pullRequest) => {
+  return pullRequest.requested_reviewers.length > 0;
+};
+
+/**
+ * This function pings and assigns all pending reviewers to a pull request.
+ * @param {import('probot').Context} context
+ * @param {import('probot').Octokit.PullsGetResponse} pullRequest
+ */
+const pingAndAssignReviewers = async (context, pullRequest) => {
```

Comment:
This function can be moved to utils and reused here and in pull request review module. You can pass in the comment and list of reviewers as argument.